code/src/index.css

```diff
@@ -1,13 +1,131 @@
+@import url('https://fonts.googleapis.com/css2?family=Assistant:wght@500&family=Sail&display=swap');
+
+html {
+  height: 100%;
+}
+
 body {
+  min-height: 100%;
+  height: 100%;
   margin: 0;
-  font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", "Roboto", "Oxygen",
-    "Ubuntu", "Cantarell", "Fira Sans", "Droid Sans", "Helvetica Neue",
-    sans-serif;
+  font-family: 'Assistant', sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  background: rgb(10,18,121);
+  background: linear-gradient(5deg, rgba(10,18,121,1) 0%, rgba(103,27,187,1) 100%);
+}
+
+h1 {
+  font-family: 'Sail', sans-serif;
+  font-size: 60px;
+  color: white;
 }
 
-code {
-  font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
-    monospace;
+header {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  margin: auto;
 }
+
+header img {
+  width: 100px;
+  align-self: flex-end;
+}
+
+button {
+  width: 90px;
+  padding: 5px;
+  background-color: #D11149;
+  border: solid 1px aliceblue;
+  border-radius: 10px;
+  align-self: center;
+  margin-top: 1em;
+  border: none;
+  font-family: 'Assistant', sans-serif;
+  color: white;
+}
+
+button:hover {
+  background-color: aliceblue;
+  color: black;
+}
+
+.label {
+  font-family: 'Assistant', sans-serif;
+  align-items: center;
+  display: flex;
+  flex-direction: column;
+  gap: 0.5rem;
+}
+
+footer {
```

Comment:
`position: absolute` here seems to cause problems on iPhone 5/SE. Maybe remove it in a media query for smaller phones? 😊